setup.py

```diff
@@ -8,14 +8,5 @@
 
 
 setup(
-    ext_modules=[
```

Comment:
Can you remove the `Extension` and `get_include` imports while you're here?

And in case you missed it above, the tests are failing on CI because we test the installed package. You need to add the following to `setup.cfg`:

```ini
[options.package_data]
gradunwarp.core.tests = data/*
```